packages/tasks/src/tasks/text-to-image/about.md

```diff
@@ -53,14 +53,21 @@ await inference.textToImage({
 
 ## Useful Resources
 
+### Model Inference
 - [Hugging Face Diffusion Models Course](https://github.com/huggingface/diffusion-models-class)
+- [Welcome aMUSEd: Efficient Text-to-Image Generation](https://huggingface.co/blog/amused)
```

Comment:
This is a bit experimental, I think the diffusers guides (at least) should come first.